packages/transformers/js/core/src/collect.rs

```diff
@@ -241,6 +245,15 @@ impl Visit for Collect {
     }
     self.in_module_this = false;
 
+    for (_key, node) in self.this_exprs.clone() {
```

Comment:
```suggestion
    for (_key, node) in &self.this_exprs {
```

Reply:
That actually doesn't work because there is a borrow to `self` and then it tries to make a mutable borrow
```js
    for (_key, node) in self.this_exprs.clone() {
		...
		self.add_bailout(...) // &mut self
	}
```